### ISSUE TITLE
Healer's Searing Brightness issues

### ISSUE THREAD

**Issue by DanielTheBard** [HabitRPG/habitica#2081]
For reference, I'm lvl 57 and my stats are: 101.5 CON, 53.5 INT, 34 PER, 36 STR
So I decided to cast Searing Brightness, this morning (redness of tasks is reduced). However, I don't think I even noticed a difference. So I decided to give myself more MP. And cast it 25 times in a row. It took a while but the biggest effect I noticed was that my + habits became blue eventually. My +/- habits turned darker blue, as well. Except for the ones that were previously orange. These didn't appear to change at all. I'm not sure if I even witnessed a change in my dailies. Most of my dailies ARE dark blue, so might be understandable. But my orange dailies didn't seem to change. I had a yellow daily, too that I THINK was yellow when I started, but I'm not positive. Also, my - habit turned blue. Which is... odd.

Basically, the spell took a LOT of casting before I noticed any real change, and it only seemed to affect the tasks that were already doing well or neutral and not the negative ones. I also have no idea if it impacted my dailies at all.

**Comment by sleepykit** [comment]
This one for me as well. As a test I reset up all of my dailies (like created brand new ones) so they're all yellow and have a 0 streak and I haven't touched them yet. I also have a few + only habits that have never been touched. Searing light did not affect them at all as far as I can see. 

I also created a habit and made it orange on purpose and then tried searing brightness and it did not become any less orange that I can see. 

Oooh... some casting it four times made a few of these yellow tasks become green (especially the ones that were already checked for the day).

**Comment by brandonjreid** [comment]
Looked into this issue.  Looking at the code and the database, it is working correctly, but it might need to be balanced better.  If your character has 200 Int, then the changes in redness are usually noticeable with only one cast.  But with a low amount of Int, it can take 10 casts before you see any difference in the redness of your tasks.  Maybe the effectiveness should be increased for lower amounts of Int?  Instead of the formula being Int \* 0.006 it could be something like (Int \* 0.005) + 0.2.

**Comment by brandonjreid** [comment]
Looking at the info on the [Trello card](https://trello.com/c/L1H6LiZ3/110-character-stats-attributes-the-class-system) it should probably be changed to use a hyperbolic function rather than a linear one.  Taking a look at Aubec's excel sheet, I tried some different variables.  I think the function should be:
1.5 \* (Int / (Int + 40))
That should make it much more effective for low amounts of int. Using it with 10 Int should change the task's value by 0.3 rather than 0.06 in the old system.  It also will ramp up the effectiveness faster and prevent the effect from getting out of control with large amounts of int since the function approaches 1.5, but never gets there.

I'm working on changing this one as well as other functions that have some issues (Pickpoket and Brutal Smash).

**Comment by lefnire** [comment]
@brandonjreid should we close this? Looks like this was part of your PR?

**Comment by brandonjreid** [comment]
Yes, this should be fixed now.
